meeting_guide/wagtail_hooks.py

```diff
@@ -69,44 +67,45 @@ class MeetingTypeAdmin(ModelAdmin):
         "display_order",
     )
     ordering = ("display_order", "type_name")
+    search_fields = ("type_name",)
 
 
-class RegionAdmin(ModelAdmin):
+class RegionAdmin(SnippetViewSet):
     model = Region
     menu_icon = "doc-full-inverse"
     empty_value_display = "-----"
     list_display = ("parent", "name")
     ordering = ("parent", "name")
-    list_filter = (RegionListFilter,)
+    # list_filter = (RegionListFilter,)

```

Comment:
So this fella gave me quite the runaround. Here's the error I was getting: 
![image](https://github.com/code4recovery/wagtail-meeting-guide/assets/4743971/bfb1f41f-c39f-4860-8417-70cca47c62ed)

I opted the keep this because, frankly, I don't know what this was originally doing. My best guess is that `SnippetViewSet`'s don't support `SimpleListFilter`.

Reply:
Hmmm, we'll have to look into this. I haven't played with `ModelAdmin` -> `SnippetViewSet` at all yet. What this does is create a top-level filter of the Region tree:

![image](https://github.com/code4recovery/wagtail-meeting-guide/assets/68164/d42fead4-b4f0-49a9-a4ec-35917ef1d842)

Reply:
Ahh ok. I'll do some tinkering to see if I can replicate that. No promises, I might get stuck on that too :P

Reply:
Commit here: https://github.com/code4recovery/wagtail-meeting-guide/pull/2/commits/370dd91093245cfda9f27fdb27bb364cc2c7ab44 

(The commit after that I re-added `.order_by()` because I forgot it in this commit)